packages/tinacms/src/toolkit/fields/plugins/mdx-field-plugin/plate/components/plate-ui/dropdown-menu.tsx

```diff
@@ -0,0 +1,180 @@
+'use client'
+
+import * as React from 'react'
```

Comment:
**question**: Is this import required, especially given that we are importing specific items on the next line?

Reply:
Yes, but also no - Cleaned it up into 1 line